src/platform/packages/shared/kbn-lens-embeddable-utils/config_builder/schema/charts/metric.ts

```diff
@@ -136,7 +136,7 @@ const metricStateSecondaryMetricOptionsSchema = schema.object({
   /**
    * Prefix
    */
-  prefix: schema.string({ meta: { description: 'Prefix' }, defaultValue: '' }),
+  prefix: schema.maybe(schema.string({ meta: { description: 'Prefix' } })),
```

Comment:
we removed most of the default values on strings that default to '', we missed this one.